### DISCUSSION TITLE
Coding Puzzle Number 0 Random Family Generator

### DISCUSSION THREAD

**Discussion by Jay-study-nildana** [Jay-study-nildana/FrontEndForStudents#16] [Coding Challenges]
Today,

Help me create a simple family generator. 

1. Father and Mother name. 
        1. Random first name and last name.
        2. Random age.
        3. Random job, with random company name.
        4. collection of random hobbies.
        5. random 10 digit phone number
1. one more children. 
        1. So, this will be a collection. 
        2. each child will have a random first name and random last name
        3. random age.
        4. random hobby
1. Random address
        1. Door number.
        2. street name
        3. PIN CODE
        4. City

The function should take a parameter, a simple number, like 10. Then, it should return a collection of 10 families, with each family object having the above things. 

Any doubts, feel free to ask here, or, on the discord server. https://discord.gg/CVugpJRBTB

Happy coding. 

Jay.

**Comment by asmallick1999** [comment]
Input: take any Integer number.
Output: return 10 objects. Where randomly create family details from the array
is it ??

**Reply by Jay-study-nildana** [reply]
@asmallick1999 

yes. that is correct. Let me try to write a sample here.

```

//random collection of family members

var family = {

    "members": [


        {
            "name": "John",
            "age": 50,

            "children": [
                {
                    "name": "Sally",
                    "age": 30
                },
                {
                    "name": "Billy",
                    "age": 28
                }
            ]
        },
        {
            "name": "Jane",
            "age": 45,

            "children": [
                {
                    "name": "Sue",
                    "age": 25
                },
                {
                    "name": "Joe",
                    "age": 22
                }
            ]
        }
    ]
};

```
but, since we are from India, perhaps, you can do with Indian names, and add more detail like jobs and things like that. 

Some data, you have to type in an array. But, majority should be generated with code, random function and for loops.
